_posts/2016-10-16-getting-started-in-scala-on-android.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+asset-type: post
+name: getting-started-in-scala-on-android
+title: Getting started in Scala on Android
```

Comment:
In fact, Carlos had a brief look last night and he pointed this out: he expected to see some actual code, given the title.